diana.navasardyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,28 @@
+inp_number1 = int(input("Enter a number: "))
+inp_number2 =  int(input("Enter a number: "))
+inp_number3 =  int(input("Enter a number: "))
```

Comment:
Try to get input in line 1.

Hint. The input can be in a one-line and it will represent one string. You can get inputs by using string operations.